README.md

```diff
@@ -7,31 +7,95 @@
 [![CI](https://github.com/code-pushup/cli/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/code-pushup/cli/actions/workflows/ci.yml?query=branch%3Amain)
 [![Codecov](https://codecov.io/gh/code-pushup/cli/branch/main/graph/badge.svg?token=Y7V489JZ4A)](https://codecov.io/gh/code-pushup/cli)
 
-🔎🔬 **Quality metrics for your software project.** 📉🔍
+<h1 align="center">🔎🔬Code quality integrations for any tool📉🔍</h1>
 
-1. ⚙️ **Configure what you want to track using your favourite tools.**
-2. 🤖 **Integrate it in your CI.**
-3. 🌈 **Visualize reports in a beautiful dashboard.**
+<h3>Code quality tools are like phone chargers. Everyone has a different plug.</h3>
+
+- **Incomplete or missing tracking** due to a mix of portals and custom tools is costly and unreliable
+- **No Standards** in data and processes leads to flaky, self-maintained zombies
```

Comment:
What is a "self-maintained zombie"? Don't you mean "unmaintained"? Self-maintained sounds like it would be a good thing :grin:

Reply:
used unmaintained